javascript/packages/stylelint-config-styled-components/.stylelintrc

```diff
@@ -1,36 +1,12 @@
 {
-  "processors": [
-    [
-      "stylelint-processor-styled-components",
-      {
-        "parserPlugins": [
-          "jsx",
-          "objectRestSpread",
-          [
-            "decorators",
-            {
-              "decoratorsBeforeExport": true
-            }
-          ],
-          "classProperties",
-          "exportExtensions",
-          "asyncGenerators",
-          "functionBind",
-          "functionSent",
-          "dynamicImport",
-          "optionalCatchBinding",
-          "optionalChaining",
-          "nullishCoalescingOperator",
-          "numericSeparator"
-        ]
-      }
-    ]
-  ],
   "extends": [
-    "@equisoft/stylelint-scss-config",
-    "stylelint-config-styled-components"
+    "@equisoft/stylelint-scss-config"
   ],
-  "plugins": ["stylelint-order"],
-  "rules": { },
-  "syntax": "scss"
+  "rules": {
+    "indentation": 4,
```

Comment:
Ça serait pas plutôt 2 pour matcher l'autre config? Ou IJ utilise l'indentation du JS?

Reply:
En fait IJ est complètement mêlé dans le css-in-js, il reformate rien dans les tagged template literals. On avait mis 4 pour que ça match l'indent du code. Mais sinon on peut mettre 4 au scss et le changer dans le .editorconfig aussi. Tu préfères quoi?

Reply:
Je mettrais 4 au scss sinon risque d'avoir des problèmes si on fait des fichiers scss dans un projet qui a styled-components.

Reply:
D'ac. Je viens de voir que tu passes stylelint avec la même config pour les fichiers scss et ts/tsx. Dans dabench on avait une config pour chaque, mais c'est sûrement plus simple d'en avoir juste une. Rendu là, je sais pas pourquoi on a 2 packages? 🤔

Reply:
Ben c'est un peu ce que je me demandais :p

Reply:
J'ai flushé le package et mergé en `stylelint-config` finalement